git_downloader.py

```diff
@@ -30,18 +38,18 @@ def main(args, loglevel):
                     if os.path.exists(output_path):
                         output_path += "-" + str(file_counter)
                     try:
-                        urllib.urlretrieve(file, output_path)
-                    except:
-                        logging.error('Error downloading %s' % file)
-            except:
-                 logging.error('Error fetching repository %s' % line)
+                        urlretrieve(file, output_path)
+                    except Exception as error:
+                        logging.error('Error downloading %s. %s' % (file, error))
```

Comment:
Nice thinking printing the error message. You can use `logging,exception('Error downloading %s', file)` instead of appending the exception error to the message (see [here](https://docs.python.org/2/library/logging.html#logging.Logger.exception)).